Makefile.am

```diff
@@ -5720,6 +5720,13 @@ dist_noinst_DATA += \
     src/tests/multihost/basic/test_sssctl_config_check.py \
     src/tests/multihost/basic/test_sudo.py \
     src/tests/multihost/basic/utils_config.py \
+	src/tests/multihost/ad/ \
+	src/tests/multihost/admultidomain/ \
```

Comment:
Please keep indentation

Reply:
And I'm not sure but most probably listing folders isn't enough, unfortunately.

Reply:
Or would `src/tests/multihost/ad/*.py` work, @pbrezina ?

Reply:
I did a paste and saw a directory and thought that was part of the original commit, so I thought it could be done.  Then I saw the diff when I pushed the file that it wasn't.

Reply:
In theory, we might be fine with just src/tests since all of it comes to dist noinst. But I have no idea if automake can handle directories recursively (or wildcards). It's worth the try.

Reply:
If not, maybe we can use `dist-hook` to copy tests to tarball https://www.gnu.org/software/automake/manual/html_node/The-dist-Hook.html